### PR TITLE
proto: rename packages from `tendermint` to `cometbft`

### PR DESCRIPTION
**Pull request by mzabaluev**
Work on #94

---

#### PR checklist

- [ ] Tests written/updated
- [ ] Changelog entry added in `.changelog` (we use [unclog](https://github.com/informalsystems/unclog) to manage our changelog)
- [ ] Updated relevant documentation (`docs/` or `spec/`) and code comments

### CODE REVIEW

**Comment by mzabaluev** [comment]
As initially pushed, this is just a mechanical rename of the packages and regeneration of the `*.pb.go` files on top of the feature branch's baseline.

This PR is currently posted only for reference.
I'm working on an attempt to introduce version suffixes for #95 on top of these changes, before integrating the generated code into the rest of the code base. Doing the massive renaming twice would be a waste of effort.

File: abci/types/types.pb.go [hunk 1]
```diff
@@ -3844,7 +3843,7 @@ func NewABCIClient(cc grpc1.ClientConn) ABCIClient {
 
 func (c *aBCIClient) Echo(ctx context.Context, in *RequestEcho, opts ...grpc.CallOption) (*ResponseEcho, error) {
 	out := new(ResponseEcho)
-	err := c.cc.Invoke(ctx, "/tendermint.abci.ABCI/Echo", in, out, opts...)
```
Comment (mzabaluev) [inline abci/types/types.pb.go:3847]: This is the gRPC protocol breakage I'm talking about in https://github.com/cometbft/cometbft/issues/94#issuecomment-1456520852

**Comment by thanethomson** [comment]
If you move all of the `*.proto` files instead of copying them and then renaming them, how do we continue to offer wire-compatible `tendermint.*` package protos to users as previously discussed?

**Comment by mzabaluev** [comment]
Superseded by #495.

**Pull request closed without merging**